### DISCUSSION TITLE
Support GitLab CI/CD components

### DISCUSSION THREAD

**Discussion by fgreinacher** [renovatebot/renovate#23424] [Suggest an Idea]
### Tell us more.

CI/CD components are a new way to reuse CI logic within GitLab, see [upstream documentation](https://docs.gitlab.com/ee/ci/components/#use-a-component-in-a-cicd-configuration).

They look very similar to other `include`s and can probably be implemented in [the existing `gitlabci-include` datasource](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/gitlabci-include/readme.md).

```yaml
include:
  - component: gitlab.com/gitlab-org/ruby-test@1.0
```

**Comment by secustor** [comment]
https://github.com/renovatebot/renovate/issues/23431

**Comment by xuhdev** [comment]
It looks like already working for me?

**Reply by secustor** [reply]
Yes, the issue above has been resolved.
